strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryFileSystemProvider.java

```diff
@@ -42,20 +42,31 @@
  *
  * @author Sergey Bespalov
  */
-public class RepositoryFileSystemProvider
+public abstract class RepositoryFileSystemProvider
         extends FileSystemProvider
 {
 
     private static final Logger logger = LoggerFactory.getLogger(RepositoryFileSystemProvider.class);
 
     private FileSystemProvider storageFileSystemProvider;
-
+    private boolean forceDelete;
+    
     public RepositoryFileSystemProvider(FileSystemProvider storageFileSystemProvider)
     {
         super();
         this.storageFileSystemProvider = storageFileSystemProvider;
     }
 
+    public boolean isForceDelete()
```

Comment:
Could this be called `allowsForceDelete`? (The same goes for the setter method).

I think I've seen this and similar cases at a few places so far.